inginious/frontend/parsable_text.py

```diff
@@ -334,6 +350,69 @@ def hdl_to_html_block(self, test_id, result, test_case, input_sample, debug_info
             html_block = html_block.replace("updateDiffBlock", "updateWaveDromBlock")
         return html_block
 
+    def client_grader_result_to_html_block(self, test):
+        """Parse each test of client grader submissions into html"""
+
+        test_functions = json.loads(test["functions"].replace("'", "\""), strict=False)
+        test_variables = json.loads(test["variables"].replace("'", "\""), strict=False)
+
+        id_num = ""
+        for i in range(0, len(test["id"])):
+            if test["id"][i].isdigit():
+                id_num += " " + test["id"][i:]
+                break
+
+        template_info = {
+            "test_id": id_num,
+            "test_grade": test["test_grade"],
+            "test_message": test["test_message"],
+            "panel_id": "collapseDebug" + str(test["id"][-1]),
```

Comment:
This part is not clear for me. Maybe:

```suggestion
            "panel_id": "collapseDebug" + id_num,
```

Reply:
Solved with https://github.com/JuezUN/INGInious/pull/504#discussion_r1136141990